src/modules/Auth/components/VerifyAccount/VerifyAccount.tsx

```diff
@@ -1,7 +1,95 @@
-import React from 'react'
+import Logo from '../../../../assets/logo.png';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { useNavigate } from 'react-router-dom';
+import { toast } from 'react-toastify';
+import { AUTHURLS, requestHeader } from './../../../../constants/URLS';
+import { EmailValidation } from '../../../../constants/validations';
+import axios from 'axios';
+
+interface FormData {
+  email: string;
+  code: string;
+}
+
 
 export default function VerifyAccount() {
+  const navigate = useNavigate();
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    watch,
+  } = useForm<FormData>();
+
+  const onSubmit: SubmitHandler<FormData> = async (data) => {
+    try {
+      const response = await axios.post<{ message: string }>(
+        AUTHURLS.verifyAccountUrl,
+        data,
+        { headers: requestHeader }
+      );
+      console.log(response.data.message);
+      navigate('/login');
+      toast.success(response.data.message);
```

Comment:
To be in the safe side, we should use optional chaining when accessing backend data to avoid runtime errors if for example, the backend changes the response structure.

Reply:
DONE